README.md

```diff
@@ -0,0 +1,48 @@
+# Hello Microverse Project
+
+> Description the project.
+
+## Built With
+
+- Html and CSS
+- Stylelint (For style linting)
+- Webhint - A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+## Getting Started
+
+To get a local copy up, CD into the root folder and run:
+
+```
+git npm install
+```
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/serengia)
+- Website: [James Serengia](https://jamesserengia.com/)
+- Twitter: [@twitterhandle](https://twitter.com/jamesserengia)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/james-serengia)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your MIT license is currently broken because the file is not present in your codebase. Kindly add the MIT license file 👍🏽

Reply:
Thank you for the feedback @vickymarz . I've made the requested changes.

Reply:
Hello @vickymarz  . I missed updating the MIT LICENSE link correctly on the readme file but it's now okay. 
You check it out. 
Thank you.